src/types.ts

```diff
@@ -22,7 +22,7 @@ export type StylexOptions = {
 
 export type UnpluginStylexOptions = {
   // compiler?: string
-  invalidExts?: string[]
+  validExts?: RegExp
```

Comment:
I think this should also support `string[]` or `string`, because only regexp maybe more complicate for some cases.

Reply:
added string[] that would allow a list of extensions like before.